src/controller/java/templates/ChipClusters-java.zapt

```diff
@@ -160,6 +160,45 @@ public class ChipClusters {
         {{/if}}
         {{/chip_attribute_list_entryTypes}}
       }
+
+      @Override
+      public String toString() {
+        StringBuilder output = new StringBuilder("");
+          {{#chip_attribute_list_entryTypes}}
+            {{#if isOptional}}
+            {{! TODO: Add support for optional types here }}
+            {{else if isNullable}}
+            {{! TODO: Add support for nullable types here }}
+            {{else if isArray}}
+            {{! TODO: Add support for lists here }}
+            {{else if isStruct}}
+            {{! TODO: Add support for structs here }}
+            {{else if (isOctetString type)}}
+            output.append("byte[] {{asLowerCamelCase name}}: [");
+            for (int i = 0; i < {{asLowerCamelCase name}}.length; i++) {
+              if (i != {{asLowerCamelCase name}}.length - 1) {
+                output.append({{asLowerCamelCase name}}[i]);
+                output.append(", ");
+              }
+              else {
+                output.append({{asLowerCamelCase name}}[i]);
+                output.append("]");
+                output.append("\n");
+              }
+            }
```

Comment:
`Arrays.toString({{asLowerCamelCase name}})` can replace the manual printing out of elements.